packages/genesys-spark-components/src/components/stable/gux-form-field/helper-components/gux-form-field-label-tooltip/gux-form-field-label-tooltip.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component, Element, h, JSX, Prop } from '@stencil/core';
+
+import { trackComponent } from '@utils/tracking/usage';
+import { GuxFormFieldTooltipVariant } from './gux-form-field-label-tooltip.types';
+import { randomHTMLId } from '@utils/dom/random-html-id';
+
+@Component({
+  styleUrl: 'gux-form-field-label-tooltip.scss',
+  tag: 'gux-form-field-label-tooltip'
+})
+export class GuxFormFieldTooltip {
+  private tooltipId: string = randomHTMLId('gux-form-field-label-tooltip');
+
+  @Element()
+  private root: HTMLElement;
+
+  @Prop()
+  variant: GuxFormFieldTooltipVariant = 'info';
+
+  @Prop()
+  screenreaderText: string;
```

Comment:
Try getting this text from the slot content to see if we can avoid the requirement for duplication.

Reply:
I removed this but changed the icon to decorative because the text would be the same and the element is essentially the same so would be kind of repeating itself.